src/Selenium2Driver.php

```diff
@@ -656,12 +656,18 @@ public function setValue($xpath, $value)
         $value = strval($value);
 
         if (in_array($elementName, array('input', 'textarea'))) {
-            $existingValueLength = strlen($element->attribute('value'));
+            $deleteSequence = Key::CONTROL . 'a' . KEY::NULL_KEY;
```

Comment:
What do you think would be a good way to detect OS? I think of adding `isMac()` method that would execute a JS `navigator.appVersion.indexOf("Mac")!==-1;` and cache the result, does this sound good?